tendrl/commons/objects/cluster_node_context/__init__.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import os
+import socket
+import uuid
+
+from tendrl.commons.etcdobj import EtcdObj
+from tendrl.commons.utils import cmd_utils
+
+from tendrl.commons import objects
+
+
+LOG = logging.getLogger(__name__)
+
+
+class ClusterNodeContext(objects.BaseObject):
+
+    def __init__(self, machine_id=None, node_id=None, fqdn=None,
+                 tags=None, status=None, *args, **kwargs):
+        super(ClusterNodeContext, self).__init__(*args, **kwargs)
+        self.value = 'clusters/%s/nodes/%s/NodeContext'
```

Comment:
Apart "value" you can get every other attribute directly from NS.node_context.$attribute

You wont need any of the below functions too apart from render()